mcs/class/System.Private.CoreLib/System/RuntimeType.cs

```diff
@@ -229,7 +229,15 @@ public override MethodInfo[] GetMethods(BindingFlags bindingAttr)
 			throw new NotImplementedException (); /* TODO */
 		}
 
-
+		internal static object CreateInstanceForAnotherGenericParameter (Type genericType, RuntimeType genericArgument)
```

Comment:
isn't this entire file dead code? I saw you removed `System/RuntimeType.cs` from `System.Private.CoreLib.csproj`